packages/i18n/fr.json

```diff
@@ -71,7 +71,7 @@
       "firstName": "Prénom",
       "username": "Nom d'utilisateur",
       "email": "Email",
-      "role": "Rôle",
+      "policy": "Groupe de stratégie",
```

Comment:
What, it is not policy!? It is user's role: ADMIN - CREATOR - MEMBER

Reply:
My bad !... I think I make a too dumb search & replace.
Fixed.